newt/repo/repo.go

```diff
@@ -191,6 +194,22 @@ func (r *Repo) patchesFilePath() string {
 		"/.patches/"
 }
 
+// Check that there is a list of submodules to clone; if the key is
+// not present, assume that all submodules should be cloned, otherwise
+// only those configured will be cloned (or none if "")
+func (r *Repo) loadSubmoduleConfig(yc *ycfg.YCfg) error {
```

Comment:
Other functions that take a `ycfg.YCfg` object take it by value, not reference.  It might make sense to change this function to match the others.

After programming in Go for several years, I still have no idea when to use pointers and when not to, so I am *not* saying pass-by-value is "better" here, just more consistent :).

Reply:
> Other functions that take a ycfg.YCfg object take it by value, not reference. It might make sense to change this function to match the others.

Done, thanks!